<launch>
<node name="trajectory_service_server" pkg="trajectory_generation" type="trajectory_server.py" output="screen" respawn="false" args="" />
</launch>